docs/product/components/breadcrumbs.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Breadcrumbs
+description: Breadcrumbs are used to provide context for the currently-viewed page.
+---
+<section class="stacks-section">
+    {% header h2 | Example %}
+    <p class="stacks-copy">Breadcrumbs show users where they are within a site’s hierarchy. Breadcrumbs help orient the user and allow for navigation to previous page levels. It can be appropriate to use a breadcrumb when:</p>
+    <ul class="stacks-list">
+        <li>The user is many levels deep</li>
+        <li>The current page does not have its own navigation</li>
+        <li>The user needs to quickly go back to the previous parent page</li>
+    </ul>
+    <p class="stacks-copy">Breadcrumbs should be the first page element—placed directly above the page’s title. The page the user is on should not appear in the breadcrumb trail.</p>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<nav class="s-breadcrumb mb8 sm:mb4">
+    <a class="s-breadcrumb--item" href="…">
+        @Svg.LogoGlyphXxs
```

Comment:
Should we add a comment that it doesn't always have to be an icon? Sometimes it can be just a label (mostly for enterprise and other communities).